src/creator/PathGraph.cpp

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2019 CFS Engineering
+ *
+ * Created: 2018 Malo Drougard <malo.drougard@protonmail.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "PathGraph.h"
+#include <algorithm>
+
+std::vector<std::string> tigl::PathGraph::GetElementsInBetween(const std::vector<std::string>& graph,
```

Comment:
Please move the functions into the CPACS Creator code, not into the TiGL Code base.

To be in TiGL, is should be more generic e.g. use templates. Also new tigl functions require a unit test.

The word "graph" is also somewhat misleading. What you actually search is a list, not a graph.

Reply:
- Unit test: added
- Rename to ListFunctions: done !
(PathGraph suggest that the graph is a path graph, but I agree that ListFunctions may be  a more intuitive name) 
- use templates option: Not done (for me, at the moment, making this functions generic will only make the syntax more heavy ) 

commit: c2d9b9084f9cf34caa645b4b95a536bd62a6541b